### PR TITLE
STSMACOM-468: Implement <ColumnManager> in the results list

### PR DESCRIPTION
**Pull request by rasmuswoelk**
This is a follow-up PR for the work done in [STSMACOM-468](https://github.com/folio-org/stripes-smart-components/pull/997).

This PR implements the new `<ColumnManager>`-component which handles persisted column visibility toggling in the results list.

Test the changes here:
https://folio-styling.testing.samhaeng.com/requests

image

### CODE REVIEW

**Comment by rasmuswoelk** [comment]
> This does not appear to work in Requests on the test site! It does in Users and Inventory, but I assume that's the older code?

Hi @Baroquem 

Thank you for your feedback!

The column manager is active on both ui-users, ui-inventory, and ui-requests on the test site so it is using the same logic for all of those modules.

Can you elaborate on what isn't working? Is it not toggling columns when you toggle the columns in the actions dropdown?

**Comment by rasmuswoelk** [comment]
@Baroquem I fixed the issue and updated the testing site – can you try it again when you have time?

**Comment by Baroquem** [comment]
Hi, @rasmuswoelk, this does seem to be working as advertised now. Thanks!

File: src/routes/RequestsRoute.js [hunk 1]
```diff
@@ -787,81 +788,88 @@ class RequestsRoute extends React.Component {
...
+                </PrintButton>
+              </>
+          }
+        </MenuSection>
+        {renderColumnsMenu}
       </>
     );
 
+    const columnManagerProps = {
+      excludeKeys: ['title'],
```
[the remark this answers is not in the document]
Comment (Baroquem) [reply src/routes/RequestsRoute.js:870]: I understand, just wanted to put that out there. Perhaps the usability testers will weigh in eventually. 🙂

**Review by Baroquem — APPROVED** [review]

**Review by mkuklis — APPROVED** [review]
Nice work @rasmuswoelk!

**Pull request merged**